exasol_integration_test_docker_environment/test/test_test_environment_db_mem_size.py

```diff
@@ -57,7 +59,7 @@ def test_invalid_db_mem_size(self):
             docker_envs = \
                 self.test_environment.spawn_docker_test_environment(self.docker_environment_name,
                                                                     ["--db-mem-size", "'999 MiB'"])
-        close_environments(*docker_envs)
+            utils.close_environments(*docker_envs)
```

Comment:
Usually, you try to generate the object for the context directly in the with statement